### PR TITLE
Update hackclub.com.yaml

### PR DESCRIPTION
**Pull request by Krish-sysadmin**

### CODE REVIEW

**Review by sampoder — CHANGES_REQUESTED** [review]
Hey! Curious, what's this sites relevance to Hack Club?

**Comment by Krish-sysadmin** [comment]
@sampoder Hey! This is just a placeholder right now to show my team that the subdomain works. We will have a proper website linked (a hackathon page).

**Comment by Krish-sysadmin** [comment]
The hackathon is for this school

File: hackclub.com.yaml [hunk 1]
```diff
@@ -220,6 +220,10 @@ alexa:
 - ttl: 1
   type: CNAME
   value: cname.vercel-dns.com.
+aloha:
+- ttl: 1
+  type: CNAME
+  value: https://aloha-college.com/en/aloha-college/.
```
Comment (sampoder) [inline hackclub.com.yaml:226]: This CNAME is configured incorrectly and will not work, CNAMEs should not include https:// and should not have a path at the end

Comment (Krish-sysadmin) [reply hackclub.com.yaml]: Fixed @sampoder

**Review by sampoder — APPROVED** [review]
Coolio, thanks!

**Pull request merged**